src/heart.c

```diff
@@ -383,12 +385,21 @@ kill_old_erlang(void)
     }
 }
 
+static void write_to_custom_error_log(int reason) {
+    FILE *fp;
+    fp = fopen(custom_error_log, "w+");
```

Comment:
Please. Please.. not to the /var/system/.. We have a directory here: /var/system/priv/log/

Reply:
Okay fair enough. Forgot about that.